### DISCUSSION TITLE
Does spark .net have any plans for .net 6

### DISCUSSION THREAD

**Discussion by Bongiboy777** [dotnet/spark#980] [General]
Does spark .net have any plans for .net 6

**Comment by MikeRys** [comment]
Support for .NET 6 is planned.

From: Bongani ***@***.***>
Sent: Tuesday, October 5, 2021 2:54 AM
To: dotnet/spark ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [dotnet/spark] Does spark .net have any plans for .net 6 (Discussion #980)


Does spark .net have any plans for .net 6

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdotnet%2Fspark%2Fdiscussions%2F980&data=04%7C01%7Cmrys%40microsoft.com%7Ce0c2f371b1914fee62f108d987e61e9a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637690244703877318%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=I61eiwwwzI8jqdAYMEeT8pKn%2BHs9D8q4IPzYliO%2BFQc%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACZXGJFOXR4U7X26U6OPTXLUFLDNFANCNFSM5FLLUZFA&data=04%7C01%7Cmrys%40microsoft.com%7Ce0c2f371b1914fee62f108d987e61e9a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637690244703887314%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=KMP1OUwXhVvWAC68j8n2hfJCwvwfIMgK30AWBN2GQdo%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cmrys%40microsoft.com%7Ce0c2f371b1914fee62f108d987e61e9a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637690244703897309%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Uznt6%2FV1dWNqZlDD%2BZnGIlU9GU54Ag6fhTvw1XQNYBk%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cmrys%40microsoft.com%7Ce0c2f371b1914fee62f108d987e61e9a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637690244703897309%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=AFm6f%2B6cCAFk9TC9LepuKXuv2nsyOsGLZRCaCHMYKb8%3D&reserved=0>.
